### ISSUE TITLE
Mobile Select - Full Screen Select Dropdown Modal For Small Touch Screens

### ISSUE THREAD

**Issue by villanus** [Semantic-Org/Semantic-UI#3534]
One of the most frustrating parts of using IOS on an iphone is the ridiculous native rolodex style select box which iOS uses for HTML inputs.   

![image](https://cloud.githubusercontent.com/assets/2213026/12083380/1c5df7e8-b257-11e5-926c-553e1b6efc6b.png)

Coming from a Windows Phone, Microsoft has really gotten this right, where when you enter a select you get a full page modal to make your selection and where on change the modal is closed. 

![image](https://cloud.githubusercontent.com/assets/2213026/12083390/5c03b3a6-b257-11e5-8bbd-f5fd67ec2c11.png)

Would it be possible using a combination of ViewPort (for smaller screens) and if touch is enabled to change the select dropdown behavior where the user gets a full page modal to make their selection?  I know this would be a little complicated, versus how the selects are currently handled in Semantic UI, Bootstrap & Foundation.   May require a combination of hidden form fields and javascript to make the selects work.    In my opinion, it would provide a great alternative to how things are done now using Semantic UI / and Native Selects.   

Looking forward to your comments & Feedback. 

FYI.
Here is a poorly made example of what I am talking about. 
http://craftpip.github.io/bootstrap-fullscreen-select/

**Comment by anorborg** [comment]
I completely agree with this and wish more ui frameworks would see this as an important UI issue. It seems that semantic-ui current implementation would be closer to being able to implement something like this. I think give how complex the select boxes can be especially if you have typeahead or search, in makes perfect sense to devote the entire UI to the operation on smaller screens. It is basically what already occurs on text controls with a virtual keyboard.

**Comment by villanus** [comment]
is this on roadmap?

**Comment by NicoBurno** [comment]
Is there any work on this feature? Or can we offer a specific PR with implementation?

**Comment by Staphwar** [comment]
Are there any work-arounds at this moment to achieve the native fall-back?
